python/ray/rllib/ppo/ppo_tf_policy.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+from ray.rllib.models.catalog import ModelCatalog
+from ray.rllib.utils.process_rollout import compute_advantages
+from ray.rllib.utils.tf_policy_graph import TFPolicyGraph
+
+
+class PPOLoss(object):
+    def __init__(self, action_space, value_targets, actions,
```

Comment:
Please don't indent like this, rather chop down the arguments to the next level. This avoids them becoming crammed towards the end of the line as the function name gets longer.